### PR TITLE
Clean up battery dispatch UI forms

### LINKED ISSUE #846: PV Battery / Merchant Plant default state of Battery Dispatch page incorrect
To replicate, create a new PV-Battery / Merchant case and go to the Battery Dispatch page. Need to check all FOM battery configurations:

image

Correct state is:

image

Workaround is to change the dispatch option to a different one and then back to Manual Dispatch.

### PR DESCRIPTION
**Pull request by cpaulgilman**
# Pull Request Template

## Description

Click through all options on battery dispatch forms for PV Battery, Generic Battery, Standalone Battery, and Fuel Cell and fix any problems with callback errors, incorrectly enabled/disabled inputs, or UI messages for FOM (Single Owner) and BTM (Commercial)

Fixes #846

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

If you have added a new compute module in a SSC pull request related to this one, be sure to check the [Process Requirements](https://github.com/NREL/SAM/wiki/Compute-modules-in-SAM).

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deploy/runtime/ui/Fuel Cell Dispatch.txt [hunk 1]
```diff
@@ -426,20 +426,55 @@ equations{ 'fuelcell_dispatch'} = define(){
...
 	enable('fuelcell_fixed_pct', value('fuelcell_dispatch_choice') == 0);
 	enable('fuelcell_dispatch_input', value('fuelcell_dispatch_choice') == 3);
 	enable('fuelcell_dispatch_input_units', value('fuelcell_dispatch_choice') == 3);
+}
 
-	toggle_dispatch();
-}; 
-
-
+function fuel_cell_manual_dispatch()
```
Comment (mjprilliman) [inline deploy/runtime/ui/Fuel Cell Dispatch.txt:452]: If you go from "Enable Battery" on the Battery Storage page to "No Battery" with "Manual Dispatch" selected for the battery dispatch option, "Manual Dispatch" can be selected for the Fuel Cell without any prompts or battery enabling. Is this intended behavior?

**Review by mjprilliman — COMMENTED** [review]
LGTM. Just one quirk with the Fuel Cell Battery manual dispatch interactions I noticed. Sorry for the delay on this review

**Comment by mjprilliman** [comment]
@cpaulgilman This can be merged

File: deploy/runtime/ui/Fuel Cell Dispatch.txt (hunk 1 above)
Comment (cpaulgilman) [reply deploy/runtime/ui/Fuel Cell Dispatch.txt]: Good catch. I fixed that in the latest commit -- the UI callback checks for manual dispatch should work for BTM and FOM.

**Pull request merged**